setup.cfg

```diff
@@ -39,3 +39,10 @@ testing =
 	# local
 	PyQt5; python_version>='3.0'
 	PySide2; python_version=='2.7'
+
+[options.entry_points]
+console_scripts = 
+	cola = cola.__main__:run
+	dag = cola.__main__:run_dag
```

Comment:
I think we've been asked to not use the generic `dag` name -- so we should drop that one.